tool/gravity/cli/plan.go

```diff
@@ -202,6 +202,10 @@ func outputOrFollowPlan(localEnv *localenv.LocalEnvironment, getPlan fsm.GetPlan
 }
 
 func followPlan(localEnv *localenv.LocalEnvironment, getPlan fsm.GetPlanFunc) error {
+	plan, err := getPlan()
```

Comment:
nit: this will now make the follow command half-resilient - before it could tolerate initial transient failure and now will fail immediately forcing the user to retry. Not sure which is better though.